### PR TITLE
Change IRC to Slack

### PR DESCRIPTION
**Pull request by janetkuo**
Follow up https://github.com/kubernetes/kubernetes/pull/16606#issuecomment-152642425
Link the users to Slack channel instead of IRC. 

cc @bgrant0607 @nikhiljindal @caesarxuchao

### CODE REVIEW

**Comment by caesarxuchao** [comment]
LGTM but will wait others to have a look.

File: _config.yml [hunk 1]
```diff
@@ -76,8 +76,8 @@ footer:
         url: https://plus.google.com/u/0/b/116512812300813784482/116512812300813784482
             
     links:
-      - label: Freenode
-        url: http://webchat.freenode.net/?channels=google-containers
+      - label: Slack
+        url: https://kubernetes.slack.com
```
Comment (bgrant0607) [inline _config.yml]: They need to sign up at http://slack.k8s.io/, so probably we should link to that instead

File: index.html [hunk 2]
```diff
@@ -67,5 +67,5 @@
...
+    text: Our Slack channel is the best way to contact our engineers and share your ideas with them.They hang out at <a href="https://kubernetes.slack.com" onclick="trackOutboundLink('https://kubernetes.slack.com'); return false;">#kubernetes-users</a> on Slack. Our user forum on <a href="http://stackoverflow.com/questions/tagged/kubernetes" onclick="trackOutboundLink('http://stackoverflow.com/questions/tagged/kubernetes'); return false;">stackoverflow</a> is a great place to go for community support<a href="http://kubernetes.io/v1.1" onclick="trackOutboundLink('http://kubernetes.io/v1.1'); return false;">.</a>
```
Comment (bgrant0607) [inline index.html]: Same comment about URL.

Also, you accidentally deleted the space after the first period.

File: _config.yml (hunk 1 above)
Comment (janetkuo) [inline _config.yml]: Fixed

File: index.html (hunk 2 above)
Comment (janetkuo) [inline index.html]: Both fixed.

**Comment by bgrant0607** [comment]
LGTM, thanks.

**Pull request merged**